bootstrap/src/main/java/org/jboss/windup/bootstrap/help/Help.java

```diff
@@ -16,6 +16,7 @@
 import org.dom4j.dom.DOMElement;
 import org.dom4j.io.SAXReader;
 import org.jboss.forge.furnace.Furnace;
+import org.jboss.forge.furnace.addons.Addon;
```

Comment:
I think this is an import added for some testing that can be removed because it's unused.

Reply:
Fixed